types/ramda/index.d.ts

```diff
@@ -1487,6 +1525,16 @@ export function of<T>(x: T): T[];
 export function omit<T, K extends string>(names: readonly K[], obj: T): Omit<T, K>;
 export function omit<K extends string>(names: readonly K[]): <T>(obj: T) => Omit<T, K>;
 
+/**
+ * Takes a binary function `f`, a unary function `g`, and two values.
+ * Applies `g` to each value, then applies the result of each to `f`.
+ * Also known as the P combinator.
+ */
+export function on<A, B, C>(f: (a: A, b: A) => B, g: (a: C) => A, a: C, b: C): B;
```

Comment:
Can we give some meaningful name to generic parameters, other than `A`, `B` and `C`?

Reply:
sure, i just copied the type names from the docs though. if you have suggestions for names though that'd be great